tests/run-make/print-request-help-stable-unstable/rmake.rs

```diff
@@ -0,0 +1,31 @@
+//! For issue-138698
+use run_make_support::{diff, rustc, similar};
+
+fn main() {
+    // Failed. stable rustc print invalid print request help
```

Comment:
Nit: this comment (and the one below) isn't super helpful, probably just remove it